scss/_utilities/_clearfix.scss

```diff
@@ -0,0 +1,19 @@
+// ==========================================================================
+// Clearfix
+// ==========================================================================
+
+
+@mixin clearfix() {
+	&:before,
+	&:after {
+		content: "";
+		display: table;
+	}
+	&:after {
+		clear: both;
+	}
+}
+
+.clearfix {
```

Comment:
Seems weird that we'd define one of these.  Do we use it a whole bunch? Doesn't seem necessary in modern design patterns IMO.

Is `overflow: auto` not descriptive enough for the cases where we want `.clearfix`.

Also, is this actually going to define `.clearfix` as a class in the output file? Seems like we'd at least want to "namespace" that?